pages/configuration/configuration-settings.mdx

```diff
@@ -198,8 +198,8 @@ This section contains the list of all other releant flags used within Memgraph.
 
 | Variable          | Description                                                                                                                                                                                                               | Type       |
 |-------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------|
-| MEMGRAPH_USER     | Username                                                                                                                                                                                                                  | `[string]` |
-| MEMGRAPH_PASSWORD | User password                                                                                                                                                                                                             | `[string]` |
+| MEMGRAPH_USER     | Specifies the username to connect to Memgraph. If the user does not already exist, Memgraph will create a new user with this username.                                                                                    | `[string]` |
+| MEMGRAPH_PASSWORD | Specifies the password for the user. If the user is being created (does not already exist), this password will be assigned. Otherwise, it is used to authenticate the connection to Memgraph.                             | `[string]` |
```

Comment:
Here, it applies the same that Matea said in other comments.

Reply:
I've updated the description of ENV variables.